### DISCUSSION TITLE
Change to `Special` affected Markdown

### DISCUSSION THREAD

**Discussion by adrian5** [2giosangmitom/nightfall.nvim#65] [General]
The fzf-lua [commit](https://github.com/2giosangmitom/nightfall.nvim/commit/1ef9c6f578d1b8e7a246f65f731f605d0c959a8a) changes the `Special` highlight for nightfall to `pale_yellow`. Do you remember what the reasoning behind this was? I don't encounter the special hl-group much.

Markdown punctuation (`>`, `-`, `*`) was affected by this and I think it looked better in the previous `light_cyan`. I'd like to decouple these if you're OK with that.

**Comment by 2giosangmitom** [comment]
I'd like each flavor to have a distinct accent color, so I changed the `Special` group. I agree to decouple these Markdown punctuation styles.
